### ISSUE TITLE
Lexicon stepwise refinement for search results

### ISSUE THREAD

**Issue by ant-tar** [modxcms/revolution#14456]
### Summary
I've found interesting behavior of step-by-step refinement of search results for MODX i18n.

### Step to reproduce
1. Please open Lexicon Management at /manager/?a=workspaces/lexicon
2. Leave default Namespace, Topic and Language values  - "core", "default" and "en"
3. Try to enter "Search" word, press "Enter"(for Windows) on keyboard, after (this is important - don't press "Clear filter" button!) - copy/paste or enter "Results" and press "Enter" again, and thirdly use the same way for third criteria - "More".

Please check this small video also:

![steps_gif](https://user-images.githubusercontent.com/2815640/53754073-e9965880-3ec3-11e9-8987-98a23717429b.gif)

### Observed behavior
You can see that results can be "clarified" step-by-step without "Clear filter" button

### Expected behavior
I'm confused, if this is feature I believe it should be documented well as I don't know that and believe many of us too :)
But if we have bug here previous results should be cleared and all lexicons from "core" namespace should be reloaded again according to new criteria after each form submit . 

### Environment
MODX 2.7.1

**Comment by JoshuaLuckers** [comment]
Good catch! I'm not sure if it's a bug or feature. If it's a bug I think it would be a great feature. If it's a feature then it should be clearer that this is a function.

**Comment by ghost** [comment]
The search feature in System Settings, Namespaces, and several other search forms does the same thing, I've always thought it was intentional.

**Comment by ghost** [comment]
A little poking around in the ExtJS docs, apparently it's a built-in feature of their search forms.

**Comment by Jako** [comment]
Kudos for posting the right combination of search query strings in the right order. 

But if you 'refine' the search the other way with 'More' => 'Results' => 'Search', you will extend the search results.

There is no refinement visible in the query, if I look into the requests/responses right.

I prefer to close this as a too early, but very nice april fool try. But maybe I did understand the issue wrong.
